util/chplenv/chpl_3p_jemalloc_configs.py

```diff
@@ -21,10 +21,15 @@ def get_jemalloc_config_file():
     return config_file
 
 @memoize
-def get_link_args():
-    jemalloc_config = get_jemalloc_config_file()
-    libs = ['-ljemalloc']
-    if os.access(jemalloc_config, os.X_OK):
-        jemalloc_libs = run_command([jemalloc_config, '--libs'])
-        libs += jemalloc_libs.split()
-    return libs
+def get_link_args(target_mem):
+    if target_mem == 'jemalloc':
+        jemalloc_config = get_jemalloc_config_file()
+        libs = ['-ljemalloc']
+        if os.access(jemalloc_config, os.X_OK):
+            jemalloc_libs = run_command([jemalloc_config, '--libs'])
+            libs += jemalloc_libs.split()
+        return libs
+    elif target_mem == 'system':
+        return ['-ljemalloc']
```

Comment:
I don't see any way to get that information out of the jemalloc package on Debian, so I don't think we have to worry about it right now. Anyway it seems to work without it.

Is there some link arg that is particularly relevant with versions >= 4.1 ?